media/org.openttd.OpenTTD.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Copyright 2018-2020 Flathub maintainers-->
+<component type="desktop-application">
+  <id>org.openttd.OpenTTD.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <name>OpenTTD</name>
+  <summary>Open source simulation game based upon Transport Tycoon Deluxe</summary>
+  <description>
+    <p>
+      OpenTTD is an open source simulation game based upon the popular
+      Microprose game "Transport Tycoon Deluxe", written by Chris Sawyer. It
+      attempts to mimic the original game as closely as possible while
+      extending it with new features.
+    </p>
+  </description>
+  <launchable type="desktop-id">openttd.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.openttd.org/screenshots/1.4-02-opengfx-1920x1200.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://www.openttd.org/screenshots/1.9-darkuk-1.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://www.openttd.org/screenshots/1.9-darkuk-2.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://www.openttd.org/screenshots/1.9-darkuk-3.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.openttd.org/</url>
+  <url type="bugtracker">https://github.com/OpenTTD/OpenTTD/issues</url>
+  <url type="help">https://wiki.openttd.org/en/Manual/</url>
+  <developer_name>The OpenTTD team</developer_name>
+  <update_contact>info@openttd.org</update_contact>
+  <provides>
+    <binary>openttd</binary>
+  </provides>
+  <releases>
+    <release version="13.4" date="2023-08-05"/>
+    <release version="13.3" date="2023-06-11"/>
+    <release version="13.2.1" date="2023-06-11"/>
+    <release version="13.2" date="2023-06-10"/>
+    <release version="13.1" date="2023-04-10"/>
+    <release version="13.0" date="2023-02-05"/>
+    <release version="12.2" date="2022-04-02"/>
+    <release version="12.1" date="2021-11-08"/>
+    <release version="12.0" date="2021-10-17"/>
+    <release version="1.11.2" date="2021-05-03"/>
+    <release version="1.11.1" date="2021-04-18"/>
+    <release version="1.11.0" date="2021-04-01"/>
+    <release version="1.10.3" date="2020-08-09"/>
+    <release version="1.10.2" date="2020-06-01"/>
+    <release version="1.10.1" date="2020-04-13"/>
+    <release version="1.10.0" date="2020-04-01"/>
+  </releases>
+  <content_rating type="oars-1.1"/>
```

Comment:
We do not have any rating done.  So what does this mean? Seems a bit weird to add.

Reply:
That can be specified at https://hughsie.github.io/oars/

I think it contains unmoderated chat in multiplayer mode, so it should have following spec:

```
 <content_rating type="oars-1.1">
    <content_attribute id="social-chat">intense</content_attribute>
  </content_rating>
```

Reply:
We tend to stay away from any rating, as we never did any official rating. So I rather just not mention it at all, than to start filling in a random rating system with the information we think is valid :)

Reply:
> So I rather just not mention it at all

Flathub requires a rating to pass validation of the AppStream metadata.

> than to start filling in a random rating system with the information we think is valid :)

The specification is versioned, used universally in the Flatpak ecosystem and fits in a simple webpage. We can call that many things, but random it is not.

Reply:
> That can be specified at https://hughsie.github.io/oars/
> 
> I think it contains unmoderated chat in multiplayer mode, so it should have following spec:
> 
> ```
>  <content_rating type="oars-1.1">
>     <content_attribute id="social-chat">intense</content_attribute>
>   </content_rating>
> ```

I have never used OpenTTD online, so I could not verify this. If that's the case, then indeed the OARS metadata need an update.

Reply:
So we gathered so far that the rating entry exists so this file can be used to pass validation on Flathub. This isn't clear from the context of this file, but that is fixable.

Sadly, the more pressing issue is that OpenTTD hasn't officially made any comments on how the game is rated. And I feel uncomfortable to do that with a line in a PR that is about a Linux-only file. I guess that should be done more clearly, and more prominent.

Further more, there are many rating systems in the world. We are confronted with most of them when publishing the game on stores. This further complicates the situation, that we have to make sure those ratings are in sync with each other. I guess it was about to happen sooner or later that we need to figure out a few of those rating systems, and put somewhere what the results are. So everyone can align on the same, and avoid exactly what happened in this short thread.

Which means we have two choices for this PR: either we remove the rating, and this PR becomes easier. Or we keep the rating, and we need to figure out (in another PR) how to rate OpenTTD for a few of these rating systems and publish the results somewhere.
This stuff is never easy :(

Reply:
It could be solved by making extra chat plugin, which would not be enabled by default. If some file existed in filesystem, it would enable in-game chat. That way parent-administrator could leave children safe with the game, if they would lack permission to install the extension (remove or install tested file). But I expect children below 13 years are not likely to play this game often. At least compile-time option would be nice for the chat.

Reply:
>  This stuff is never easy :(

Hence why this [generator](https://hughsie.github.io/oars/generate.html) exists: to save time from upstream developers.

Now, that's a lot of text to say "we don't care". So let's not lose further time on this.

Reply:
What is the goal of the rating? How pedantic does one need to be when making the rating?

I would say you forgot moderate money-gambling; for example attempting to bribe local authorities which is practically gambling with virtual money on a successful bribe.
And drugs-alcohol due to FIRS having alcohol as cargo, which is essentially consumed by the human units.
And drugs-tobacco due to Industries of the Caribbean having tobacco as cargo. (https://newgrf-specs.tt-wiki.net/wiki/CargoTypes)
And violence-fantasy as you can deliberately crash trains/vehicles and kill many human units.
And sex-nudity as Long road vehicle 4 has vehicle with imagery that could be recognised as naked women, so there's even a version with those images cut (http://george.zernebok.net/newgrf/downloads.html)

And to be honest... intense desecration is feasible with a NewGRF. Just have an industry that accepts passengers, when passengers arrive they are put in a pen and some animals start feasting on them until they are gone. Does that mean OpenTTD should be rated like that, since it is possible with readily downloadable graphics sets?

In any case, I would rather not self-rate the application as it will open a huge can of worms of people who see things differently. In which the developers are caught in the middle as they cannot please both sides, so if it doesn't need to be provided for anything that the OpenTTD developers provide I rather not add this point of contention.

Reply:
> It could be solved by making extra chat plugin, which would not be enabled by default. If some file existed in filesystem, it would enable in-game chat. That way parent-administrator could leave children safe with the game, if they would lack permission to install the extension (remove or install tested file). But I expect children below 13 years are not likely to play this game often. At least compile-time option would be nice for the chat.

I never thought about that, but you make a very good point. And it might surprise you how young our players can be .. at least, some of the older players always pride themselves with telling their N-year-old is playing OpenTTD :P Tnx for this sharp remark, it is a bit of an eye-opener (might sound silly to some, but I am just being honest here).

Reply:
Of course local games are safe for any children and I expect that would be most common. For online play it would be nice, if chat could be hidden on online game. Maybe just messages from the server shown. You can still make some rude images or words using railroads, but I expect that would not allow unwanted chat with the kid. Current unchecked chat allows sexual harassment among other thing unfortunately.

Reply:
> What is the goal of the rating? How pedantic does one need to be when making the rating?

I think this is a best effort thing. What people responsible for the release agree upon.
> 
> I would say you forgot moderate money-gambling; for example attempting to bribe local authorities which is practically gambling with virtual money on a successful bribe. And drugs-alcohol due to FIRS having alcohol as cargo, which is essentially consumed by the human units. And drugs-tobacco due to Industries of the Caribbean having tobacco as cargo. (https://newgrf-specs.tt-wiki.net/wiki/CargoTypes) And violence-fantasy as you can deliberately crash trains/vehicles and kill many human units. And sex-nudity as Long road vehicle 4 has vehicle with imagery that could be recognised as naked women, so there's even a version with those images cut (http://george.zernebok.net/newgrf/downloads.html)

Well, good point. With online content made by 3rd parties and downloaded in the game, it might require extensions to declare how they are rated themselves. I am not aware there is a way to declare account a child one, with age limit, on a Linux desktop. Similar to what Android offers. Might be good to have downloading of new extensions disabled by administrator decision as well.
> 
> And to be honest... intense desecration is feasible with a NewGRF. Just have an industry that accepts passengers, when passengers arrive they are put in a pen and some animals start feasting on them until they are gone. Does that mean OpenTTD should be rated like that, since it is possible with readily downloadable graphics sets?
That would be why even Steam disables extensions in family locked accounts. I would say we should rate what is contained in the game itself, not their extensions. But some future decision might design some concept, how the game should/can protect underage players.
> 
> In any case, I would rather not self-rate the application as it will open a huge can of worms of people who see things differently. In which the developers are caught in the middle as they cannot please both sides, so if it doesn't need to be provided for anything that the OpenTTD developers provide I rather not add this point of contention.

This is not an option already. Flatpak already requires some definition for a reason. I am quite sure Steam does as well, even though there is already OpenTTD without any adult specifications. I would say every current app store has similar thing. Pretending we do not do won't work. Stating explicitly it is only for adults would, but should not be necessary.